### PR TITLE
The wrong UI is applied for 'sync code' section when light mode is en…

### LINKED ISSUE #19064: The wrong UI is applied for 'sync code' section when light mode is enabled
### Steps:
1. Enable the light theme
2. Go to Profile -> Syncing -> tap [+]
3. tap 'Reveal the code' button
4. Check the UI for authorization drawer
5. Enter a valid Auth password
6. Check UI for 'sync code' section 

### Actual result:
The wrong UI is applied for authorization drawer
image

The wrong UI is applied for 'sync code' section when light mode is enabled
image

### Expected result:
Correct UI is applied for authorization drawer
https://www.figma.com/file/V6nlpAWIf2e1XU8RJ9yQPe/Syncing-for-Mobile?type=design&node-id=1112-139040&mode=design&t=XMbD52sLtsZTyPw4-0

Correct UI is applied for 'sync code' section
https://www.figma.com/file/V6nlpAWIf2e1XU8RJ9yQPe/Syncing-for-Mobile?type=design&node-id=1124-127864&mode=design&t=XMbD52sLtsZTyPw4-0

### Devices:
IOS, Android

### ENV:
[29 Feb 2024]

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #19064

### CODE REVIEW

**Review by ibrkhalil — APPROVED** [review]

File: src/quo/components/inputs/input/view.cljs [hunk 1]
```diff
@@ -55,16 +55,17 @@
 
 (def ^:private custom-props
   "Custom properties that must be removed from properties map passed to InputText."
-  [:type :blur? :theme :error? :right-icon :left-icon :disabled? :small? :button
+  [:type :blur? :error? :right-icon :left-icon :disabled? :small? :button
    :label :char-limit :on-char-limit-reach :icon-name :multiline? :on-focus :on-blur
    :container-style])
 
 (defn- base-input
-  [{:keys [blur? theme error? right-icon left-icon disabled? small? button
```
Comment (siddarthkay) [inline src/quo/components/inputs/input/view.cljs:63]: if we're removing `theme` prop from here and relying on theme value from `(quo.theme/use-theme-value)` should we also modify usage of `input` component where theme value is directly passed?
example : https://github.com/status-im/status-mobile/blob/f788d7bc85349f26115b3e052a437e45db0d6bc4/src/status_im/contexts/profile/edit/name/view.cljs#L47-L56

Comment (flexsurfer) [reply src/quo/components/inputs/input/view.cljs:63]: that's actually interesting, it should be override-theme i guess

Comment (flexsurfer) [reply src/quo/components/inputs/input/view.cljs:63]: oh that's probably was implemented wrong because of this bug :) so someone just passed :dark :)

Comment (flexsurfer) [reply src/quo/components/inputs/input/view.cljs:63]: fixed thanks

**Review by siddarthkay — APPROVED** [review]

**Pull request merged**